### PR TITLE
updated box.md

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [*] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/box.md [hunk 1]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
-    4. **Specify a timeout for your command**: Active the checkbox and select **60 Minutes**
-    5. **Command** (Required)`/bin/bash` (specify the correct path on your system)
-    6. **Script** (Required) Use the path to `sumojanus`, such as: `/home/ubuntu/sumojanus-box/bin/SumoJanus_Box.bash`. Do not select “Type the script to execute.”
-    7. **Working Directory.** `/home/ubuntu/sumojanus-box`.
-3. Click **Save**.
+### Authentication
+
+You'll need a Box App Key, App Secret, and Access Code to provide to Sumo Logic.
+
+The following steps must be followed to get the credentials:
```
Comment (stacykor) [inline docs/integrations/saas-cloud/box.md:90]: Complete the following steps to get the credentials:

File: docs/integrations/saas-cloud/box.md [hunk 2]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
+4. Go to the `Configuration` and change `App Access Level` to `App + Enterprise Access` and enable `Manage Enterprise properties` in `Application Scopes` and save changes as shown below.<br/><img src={useBaseUrl('img/send-data/box-source4.png')} alt="Box" /> <br/><br/><img src={useBaseUrl('img/send-data/box-source5.png')} alt="Box" /> <br/><img src={useBaseUrl('img/send-data/box-source6.png')} alt="Box" />
+5. Authorize your app by following these steps [Authorize](https://developer.box.com/guides/authentication/jwt/jwt-setup/#app-authorization).
+
+
+### States
+
+A Box Source tracks errors, reports its health, and start-up progress. You’re informed, in real-time, if the Source is having trouble connecting, if there's an error requiring user action, or if it is healthy and collecting by utilizing [Health Events](/docs/manage/Health-Events).
+
+A Box Source goes through the following states when created:
+1. **Pending**: Once the Source is submitted it is validated, stored, and placed in a **Pending** state.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/box.md:103]: **Pending**. Once the Source is submitted it is validated, stored, and placed in a **Pending** state.

File: docs/integrations/saas-cloud/box.md [hunk 3]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
+2. **Started**: A collection task is created on the Hosted Collector.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/box.md:104]: **Started**. A collection task is created on the Hosted Collector.

File: docs/integrations/saas-cloud/box.md [hunk 4]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
+3. **Initialized**: The task configuration is complete in Sumo Logic.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/box.md:105]: **Initialized**. The task configuration is complete in Sumo Logic.

File: docs/integrations/saas-cloud/box.md [hunk 5]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
+4. **Authenticated**: The Source successfully authenticated with Box.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/box.md:106]: **Authenticated**. The Source successfully authenticated with Box.

File: docs/integrations/saas-cloud/box.md [hunk 6]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
+  <tr>
+   <td>category
+   </td>
+   <td>String
+   </td>
+   <td>No
+   </td>
+   <td>null
+   </td>
+   <td>Type a category of the source. This value is assigned to the <a href="/docs/search/Get-Started-with-Search/Search-Basics/Built-in-Metadata">metadata</a> field <code>_sourceCategory</code>. See <a href="/docs/send-data/best-practices#good-and-bad-source-categories">best practices</a> for details.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/box.md:345]: <td>Type a category for the source. This value is assigned to the <a href="/docs/search/Get-Started-with-Search/Search-Basics/Built-in-Metadata">metadata</a> field <code>_sourceCategory</code>.

**Review by stacykor — CHANGES_REQUESTED** [review]
Small stuff.

File: docs/integrations/saas-cloud/box.md (hunk 1 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: done

File: docs/integrations/saas-cloud/box.md (hunk 3 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: done

File: docs/integrations/saas-cloud/box.md (hunk 5 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: done

File: docs/integrations/saas-cloud/box.md (hunk 6 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md:345]: What's the change here?

File: docs/integrations/saas-cloud/box.md (hunk 2 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: done

File: docs/integrations/saas-cloud/box.md (hunk 4 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: done

File: docs/integrations/saas-cloud/box.md (hunk 6 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md:345]: Screen Shot 1401-08-30 at 22 57 26

**Comment by JainM6** [comment]
> Small stuff.
@stacykor: Thanks for the suggestions. I have fixed all except the metadata fields --line 345. Unable to see the change.

File: docs/integrations/saas-cloud/box.md (hunk 6 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/box.md:345]: It looks like her change was:

Type a category _**for**_ the source.

**Review by stacykor — COMMENTED** [review]
for

File: docs/integrations/saas-cloud/box.md [hunk 7]
```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
...
+   <td>&#123;event_type&#125;
+   </td>
+  </tr>
+</table>
+
+
+1. (Optional) **Fields**. Click the **+Add** link to add custom log metadata [Fields](/docs/manage/fields).
+    * Define the fields you want to associate, each field needs a name (key) and value.
+        * A green circle with a checkmark is shown when the field exists and is enabled in the Fields table schema.
+        * An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/box.md:172]: Green circle & orange triangle thumbnails needed here. You can refer to most Integrations or Source docs.

**Review by kimsauce — DISMISSED** [review]
Lines 22-66: I think these need to be split into two separate JSON code blocks.

File: docs/integrations/saas-cloud/box.md (hunk 6 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: @kimsauce: Thanks for helping me to understand the change. 
@stacykor: I believe "of" is more appropriate here as it says about "some connection to object" here (category and source). We use "for" to show reasons or purpose. Let me know if we can agree on this and close it.

File: docs/integrations/saas-cloud/box.md (hunk 7 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/box.md:172]: done

File: docs/integrations/saas-cloud/box.md (hunk 6 above)
Comment (stacykor) [reply docs/integrations/saas-cloud/box.md]: No. You need to look at how you're using the definite article. 
"Type the category of the source".  Acceptable 
"Type a category for the source." Which category does the source belong to. 

Pick one but you can't say Type a category of the source. It's its indefinite.

Comment (JainM6) [reply docs/integrations/saas-cloud/box.md]: The category is not for the source, it is "of" the source. 
The first one is acceptable, however, we should avoid using 2 "the" within the same sentence in such a short phrase. For now, I am keeping the first one.

**Comment by JainM6** [comment]
> Small stuff.

Done.

**Comment by kimsauce** [comment]
@duchatran @carlos-castillo-jask , would you kindly review lines 22-66 here? It appears to be two separate queries erroneously merged into one sample. Should they be split into two code blocks?

**Review by nmohantyctr — DISMISSED** [review]

File: docs/integrations/saas-cloud/box.md (hunk 7 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/box.md:172]: I don't see them -- be sure to push the commit to this PR once you've made the change locally.

Comment (kimsauce) [inline docs/integrations/saas-cloud/box.md:172]:
```suggestion
        * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a checkmark is shown when the field exists and is enabled in the Fields table schema.
        * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
```

**Review by stacykor — APPROVED** [review]

**Comment by JainM6** [comment]
Creating a new PR so closing this one.

**Pull request closed without merging**